pandas/core/groupby/groupby.py

```diff
@@ -3303,6 +3331,20 @@ def ohlc(self) -> DataFrame:
             open high  low close open high  low close
         CAC  2.3  4.5  1.0   1.0  9.0  9.4  1.0   1.0
         SPX  1.2  8.9  1.2   2.0  3.4  8.8  3.4   8.2
+
+        For Resampler:
+
+        >>> ser = pd.Series([1, 3, 2, 4, 3, 5],
+        ...                 index=pd.DatetimeIndex(['2023-01-01',
+        ...                                         '2023-01-10',
+        ...                                         '2023-01-15',
+        ...                                         '2023-02-01',
+        ...                                         '2023-02-10',
+        ...                                         '2023-02-15']))
+        >>> print(ser.resample('MS').ohlc())
```

Comment:
I think we don't need the print statement? can be fixed up as part of a future PR anyway

Reply:
Thanks. (It looked weird in Jupyter without it)